### DISCUSSION TITLE
ERROR: numpy/sympy/scipy not available! Disentangling disabled!!

### DISCUSSION THREAD

**Discussion by valenoe** [Kinggerm/GetOrganelle#234] [Q&A]
Hello
when run this ./get_organelle_from_reads.py -1 test_1.fastq -2 test_2.fastq -R 10 -t 8 -F animal_mt -o ranita4_mt_out

I get this error 
ERROR: numpy/sympy/scipy not available! Disentangling disabled!!

I'm not sure how the program was installed and I don't know how to solve it, can you help me?
[get_org.log.txt](https://github.com/Kinggerm/GetOrganelle/files/10571359/get_org.log.txt)

**Comment by Kinggerm** [comment]
Please follow the guidelines here to reinstall GetOrganelle https://github.com/Kinggerm/GetOrganelle/wiki/Installation

**Comment by L1curry** [comment]
这个问题解决了吗？是因为Python中相关包没有安装吗？有没有不重新安装GetOrganelle解决问题的办法，我显示的是这个：
WARNING: sympy not available! Disentangling disabled!
